bin/resalloc-aws-new

```diff
@@ -265,9 +265,11 @@ InstanceInterruptionBehavior=terminate,SpotInstanceType=one-time}"
     start_opts+=( --instance-market-options "$market_options" )
 fi
 
+# Use gp3 volume for the root filesystem
+start_opts+=( --block-device-mappings "DeviceName=/dev/sda1,Ebs={VolumeType=gp3}" )
 
 if test -n "$opt_additional_volume_size"; then
-    start_opts+=( --block-device-mappings "DeviceName=/dev/sdd,Ebs={DeleteOnTermination=true,VolumeSize=$opt_additional_volume_size,VolumeType=gp2}" )
+    start_opts+=( --block-device-mappings "DeviceName=/dev/sdd,Ebs={DeleteOnTermination=true,VolumeSize=$opt_additional_volume_size,VolumeType=gp3}" )
```

Comment:
I am a bit confused. The root filesystem is now gp3 and works fine. But this line either isn't used or doesn't work.

For aarch64 builders we have two volumes:



Volume ID | Device name | Volume size (GiB) | Attachment status
-- | -- | -- | --
redacted | /dev/sda1 | 6 | Attached
redacted | /dev/sdd | 160 | Attached

And the second volume is still gp2. Do you know how the second volume is created? I cannot find it anywhere.

Reply:
Not sure :-( this should work, especially for the "additional" disk.

Reply:
I mean, this should be the place where we create the additional_volume.

Reply:
I updated the PR. This works and sets gp3 for both disks.

All the confusion came from the fact, that this `if test -n "$opt_additional_volume_size"; then` condition never happened, the additional disk for aarch64 instance is added elsewhere. Maybe it is configured in its AMI?

I needed to update the `/usr/local/bin/copr-resalloc-aws-new-aarch64` and add `--additional-volume-size 160` so that we jump to the condition and configure the additional disk.